styles/index.css

```diff
@@ -0,0 +1,41 @@
+.container {
+display: flex;
+flex-direction: row;
+flex-wrap: wrap;
+justify-content: space-around;
+align-items: center;
+text-align: center;
+font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+height: 100vh;
+color: white;
+width: 800px;
+height: 400px;
+background-color: black;
+background: linear-gradient(
+    to bottom,
+    rgb(123, 209, 159),
+    rgb(81, 184, 143),
+    rgb(108, 144, 229),
+    rgb(81, 65, 200),
+    rgb(109, 34, 221),
+    rgb(82, 19, 134),
+    rgb(58, 12, 80)
+);
+}
+
+div {
+width: 150px;
+height: 150px;
+text-align: center;
+background-color: hotpink;
+}
+div {
+width: 300px;
+height: 150px;
+text-align: center;
+background-color: transparent;
+}
+#navigation {
+width: 750px;
+background-color: transparent;
+} 
```

Comment:
Looks good, just need the body of each rule to be indented to the right one level and there should be a space between the closing bracket of a rule and the beginning of the next rule.